README.RDAP.md

```diff
@@ -34,45 +44,83 @@ Entity Primary Key can match multiple objects
 ---------------------------------------------
 If an entity primary key matches more than one object, a 500 Internal Server Error is returned.
 
+This can not be easily fixed because the same key can be used in multiple different object types: mntner and
+person/role. So it is not clear which object must be returned for a single object request.
+
 For example: https://rdap.db.ripe.net/entity/KR4422-RIPE
 
 Related Contact information is Filtered
 ---------------------------------------
-Any related contact entities ("technical","administrative" etc.) have filtered contact information, i.e. "e-mail" 
+Some related contact entities ("technical","administrative" etc.) have filtered contact information, i.e. "e-mail"
 and "notify" values are not included. This was done to avoid blocking clients for inadvertently querying excessively for personal data.
 
-A workaround is to query for each entity separately using the contact's nic-hdl, and the unfiltered information is returned (although a limit for personal data does apply).
+For /entity/ request "e-mail" will be enabled and the clients requesting those service will need to comply with the daily limit
+according to the AUP: https://www.ripe.net/manage-ips-and-asns/db/support/documentation/ripe-database-acceptable-use
```

Comment:
Did the URL get chopped up?

Reply:
Also the first mention of an acronym should spell it out, e.g. "Acceptable Use Policy (AUP)" then any subsequent mentions can use the abbreviation, i.e. "AUP".